python/runtime/temp_file.py

```diff
@@ -19,7 +19,7 @@
 # NOTE: Python 2 does not have tempfile.TemporaryDirectory. To unify the code
 # of Python 2 and 3, we make the following class.
 class TemporaryDirectory(object):
-    def __init__(self, as_cwd=False, suffix=None, prefix=None, dir=None):
+    def __init__(self, as_cwd=False, suffix="", prefix="", dir=""):
```

Comment:
Why change this?

Reply:
When running on PAI using Python 2.7.x, below `suffix + some_str` may cause error since `suffix` is None type by default.